plugins/cisco_firepower_management_center/requirements.txt

```diff
@@ -1,7 +1,7 @@
 # List third-party dependencies here, separated by newlines. 
 # All dependencies must be version-pinned, eg. requests==1.2.0
 # See: https://pip.pypa.io/en/stable/user_guide/#requirements-files
-fmcapi==20211214.0
-validators==0.18.2
-pyOpenSSL==23.0.0
-parameterized==0.8.1
\ No newline at end of file
+fmcapi==20231211.0
+pyOpenSSL==23.3.0
+validators==0.22.0
+parameterized==0.9.0
```

Comment:
Just checking if we need to pin cryptography here?

Reply:
Rest of the changes look good, happy to approve once this comment is resolved. Looking at the dependancies for pyopenssl the snyk vulns should be addressed now? But could be worth pinning as Adam mentioned?

Reply:
Agreed, think cryptography will need pinned to 41.0.2 or the vulnerability will remain.

Reply:
The latest `pyOpenSSL==23.3.0` installs latest version of `cryptography==41.0.7`. This is why I haven't pinned it there 😄